### PR TITLE
Validates and clear form

### PR DESCRIPTION
**Pull request by chantal66**
- validates names and calories with custom error messages.
- clears the form after submitting data

### CODE REVIEW

File: foods.html [hunk 1]
```diff
@@ -20,12 +20,12 @@ <h2 class="food-heading heading">Add a New Food</h2>
       <form class="form-food">
           <div class="form-food-name">
               <label class="name-label" >Food Name:
-                  <input class="name-input">
+                  <input class="name-input" type="text" required oninvalid="this.setCustomValidity('Please enter a food name')">
```
Comment (nickedwards109) [inline foods.html]: Nice!

File: lib/models/food.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ class Food {
               const foodRowTemplate = HTMLHelper.toHtml(foodName, foodCalorie, id);
               $('.all-foods tr:first').after(foodRowTemplate);
             })
+            .then(HTMLHelper.resetForm())
```
Comment (nickedwards109) [inline lib/models/food.js:39]: @chantal66 This is great!

**Comment by nickedwards109** [comment]
@chantal66 This all looks really great. I am learning from you and your code!

**Comment by chantal66** [comment]
Thank you so much for those kind words

**Pull request merged**